client/slo_test.go

```diff
@@ -41,60 +42,88 @@ func TestSLOs(t *testing.T) {
 			TimePeriodDays:   30,
 			TargetPerMillion: 995000,
 			SLI:              client.SLIRef{Alias: sli.Alias},
+			Tags: []client.Tag{
+				{Key: "color", Value: "blue"},
+				{Key: "team", Value: "a-team"},
+			},
 		}
 		slo, err = c.SLOs.Create(ctx, dataset, data)
-
 		require.NoError(t, err, "unable to create SLO")
-		assert.NotNil(t, slo.ID, "SLO ID is empty")
```

Comment:
this test was written in a hacky-lazy way (my bad), so instead of equating two full objects it's now doing more assertions of the various properties